constitution.tex

```diff
@@ -770,6 +770,16 @@
 \item Candidates must be Active Members
 \end{enumerate}
 
+\asubsection{Waiving of Requirements}
+\begin{enumerate}
+  \item The following requirements may be waived as decided by a Two-Thirds Immediate Vote as described in \ref{Two-Thirds} and \ref{Immediate Vote}:
+    \begin{enumerate}
+      \item Candidates must have at least one full semester of House membership as an Active Member
+      \item Candidates must reside on floor during the term of office
+    \end{enumerate}
+  \item This process waives the enumerated requirements for a given election for all candidates on a per-directorship basis
```

Comment:
`per-directorship` isn't well defined. Perhaps `in the selection process for a single vacant voting executive board position or for chair` or similar?

Reply:
Could probably make that more concise and accurate, like `...given election for a single executive board or chair position` since elections aren't always held when a seat is vacant, like at the end of a school year

Reply:
```tex
  \item This process operates for one requirement at a time
  \item This process operates for one voting executive board position or the Chairperson at a time
  \item This process operates on the current election only
``` thoughts?